app/models/concerns/with_assignments.rb

```diff
@@ -38,10 +38,19 @@ def last_submission_date_for(user)
     assignment_for(user).try(&:updated_at)
   end
 
+  def last_persisted_submission_id_for(user)
+    submit_solution!(user, content: '') if assignment_for(user).blank?
```

Comment:
This name is misleading. It looks like it would just do a query, but it is actually sending a blank submission. 

Be aware that this code is also hitting runners :fearful:. I don't like that.